tidb-cloud/ticloud-connect.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ticloud connect
+summary: The reference of `ticloud connect`.
+---
+
+# ticloud connect
+
+Connect to a TiDB Cloud cluster:
+
+```shell
+ticloud connect [flags]
+```
+
+> **Note:**
+>
+> The connection forces the ANSI SQL mode for the session.
+You can get more information about the ANSI SQL mode from [ANSI SQL mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sqlmode_ansi)
+
+## Examples
+
+Connect to a TiDB Cloud cluster in interactive mode:
+
+```shell
+ticloud connect
+```
+
+Use the default user to connect to a TiDB Cloud cluster in non-interactive mode:
```

Comment:
What is the default user? The root user?

Reply:
For dedicated, it is root. For serverless, it is root with prefixes, such as "SfgdsWRdfs342.root"

Reply:
If they use CLI to create the cluster, there is a step to set the password for the root user.
If they want to reset the password, they need to go to the TiDB Cloud console.